ports/soundtouch/vcpkg.json

```
@@ -0,0 +1,14 @@
+{
+  "name": "soundtouch",
+  "version-string": "2.0.0",
```

```suggestion
  "version-semver": "2.0.0",
```
This might be served by [version-semver](https://github.com/microsoft/vcpkg/blob/master/docs/users/versioning.md#version-semver).